app/src/main/java/com/it3048/mobiledeviceproject/MainViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.it3048.mobiledeviceproject
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.it3048.mobiledeviceproject.dto.Calendar
+import java.util.*
+
+
+class MainViewModel {
+    var calendar: MutableLiveData<ArrayList<Calendar>> = MutableLiveData<ArrayList<Calendar>>(){
+
```

Comment:
This appears to be an incomplete feature.  In a code review, focus on reducing technical debt of existing code.  Avoid adding new, incomplete functionality.
1) Only features that meet the Definition of Done should be merged to Master.
2) Unused code is clutter, which increases technical debt.